### ISSUE TITLE
error in system_alerts example 

### ISSUE THREAD

**Issue by albertoabellagarcia** [MobilityData/gbfs#377]
The example of payload for the system_alerts data model does not validate against its schema, _start_ and _end_ should be literal numbers and not strings as provided. 
Additionally in other examples of data models  (sorry I did not take notes of which ones) failed validations against their schemas because the _version_ attribute has not the enumeration value "3.0"

**Comment by heidiguenin** [comment]
We [moved this issue](https://github.com/MobilityData/gbfs-validator/issues/46) to the gbfs validator repo, so I'm closing it here.

**Comment by isabelle-dr** [comment]
Hi @albertoabellagarcia, thanks for opening this issue! 
I'm looking into and I can't find an error in the JSON schemas related to this. 
- the `start` and `end` in `system_alerts` are modelled this way:
```json
  "start": {
      "description": "Start time of the alert.",
      "type": "number",
      "minimum": 1450155600
    },
    "end": {
      "description": "End time of the alert.",
      "type": "number",
      "minimum": 1450155600
    }
```
This is how the `timestamp` types are modelled in the JSON Schemas

- The `version` attribute of all `gbfs_versions.json` files have the value 3.0. 

Could you explain those errors a little more? Is it an error in the validator itself?
Thanks

**Comment by albertoabellagarcia** [comment]
If you take the [schema for system_alerts](https://github.com/MobilityData/gbfs-json-schema/blob/master/system_alerts.json) and the payload in the [examples](https://github.com/NABSA/gbfs/blob/v2.2/gbfs.md#system_alertsjson) is wrong because the figures are quoted (therefore it is a string) while they should be unquoted to be a number. (for times subproperty of data->alerts)

**Comment by isabelle-dr** [comment]
Oh okay I see :)
This has been fixed since the v2.2 release, see [the master branch](https://github.com/NABSA/gbfs/blob/master/gbfs.md#system_alertsjson).
